util/configuration.py

```diff
@@ -1,6 +1,6 @@
 from .mielib.custominput import choice_input, regex_input, range_input, \
     int_input
-from .cron import CronDate, CronFrequency
```

Comment:
Why did you remove these imports?

Reply:
Are conflicts with another import? Because it might cause an issue if I can't use those moving forward.

Reply:
Those should not have been removed... They should have been commented. They were causing an issue with my setup because they are not finished and unused in that file.